src/Maui/Prism.Maui/Navigation/INavigationServiceExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static class INavigationServiceExtensions
     /// </summary>
     /// <returns><see cref="INavigationResult"/> indicating whether the request was successful or if there was an encountered <see cref="Exception"/>.</returns>
     public static Task<INavigationResult> GoBackAsync(this INavigationService navigationService) =>
-        navigationService.GoBackAsync(null);
+        navigationService.GoBackAsync((INavigationParameters)null);
```

Comment:
we do not want to be passing a null value. Create a new instance of NavigationParameters

Reply:
I modified it to pass an instantiated NavigationParameters.  
https://github.com/PrismLibrary/Prism/pull/3048/files#diff-df76ce41111618523c5ba67f2a5750376a834eba0a35dc12babad363edac96faR23